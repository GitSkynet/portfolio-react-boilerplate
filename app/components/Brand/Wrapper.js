import styled from 'styled-components';

const Wrapper = styled.div`
  padding: 0.5rem;
  padding-bottom: 0;
`;

export default Wrapper;
