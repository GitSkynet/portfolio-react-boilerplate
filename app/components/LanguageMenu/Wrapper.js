import styled from 'styled-components';

const Wrapper = styled.div`
  align-self: center;
  position: relative;
`;

export default Wrapper;
