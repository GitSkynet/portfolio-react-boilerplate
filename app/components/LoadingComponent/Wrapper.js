import styled from 'styled-components';

const Wrapper = styled.article`
  align-self: center;
  margin: 2em auto;
  position: relative;
  width: 100%;
  text-align: center;
`;

export default Wrapper;
