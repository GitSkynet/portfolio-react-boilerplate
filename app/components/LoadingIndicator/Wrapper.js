import styled from 'styled-components';

const Wrapper = styled.div`
  align-self: center;
  height: 40px;
  margin: 2em auto;
  position: relative;
  width: 40px;
`;

export default Wrapper;
