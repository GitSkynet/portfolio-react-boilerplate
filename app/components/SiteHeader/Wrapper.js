import styled from 'styled-components';

const Wrapper = styled.div`
  display: flex;
  width: 100%;
  border-bottom: 0.5vh solid #465a65;
`;

export default Wrapper;
