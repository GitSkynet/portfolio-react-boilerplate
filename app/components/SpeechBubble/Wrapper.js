import styled from 'styled-components';
import { rem } from 'polished';

import Bubble from './Bubble';

const Wrapper = styled.div`
  margin-bottom: 9vh;
  padding: 1vh 1vw;
  width: 100%;
  filter: drop-shadow(0 0 10px grey);

  @media (min-width: 1008px) {
    max-width: ${rem('1008px')};
  }

  ${Bubble}

  p {
    &:not(:first-child) {
      margin-top: 1.5rem;
    }
  }
`;

export default Wrapper;
