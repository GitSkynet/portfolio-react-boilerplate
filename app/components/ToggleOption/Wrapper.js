import styled from 'styled-components';
import { rem } from 'polished';

const Wrapper = styled.div`
  padding: ${rem(7)};
  background-color: #797978;
`;
export default Wrapper;
