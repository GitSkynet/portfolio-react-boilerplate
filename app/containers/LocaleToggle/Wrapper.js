import styled from 'styled-components';

const Wrapper = styled.aside`
  margin-left: auto;
`;

export default Wrapper;
