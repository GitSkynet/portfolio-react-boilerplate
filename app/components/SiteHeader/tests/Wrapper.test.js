import React from 'react';
import { shallow } from 'enzyme';

import Wrapper from '../Wrapper';

describe('<Wrapper />', () => {
  it('should render and match the snapshot', () => {
    const renderedComponent = shallow(<Wrapper />);
    expect(renderedComponent).toMatchSnapshot();
  });

  it('should adopt the "backgroundColor" property', () => {
    const color = '#bada55';
    const renderedComponent = shallow(<Wrapper borderColor={color} />);
    expect(renderedComponent).toMatchSnapshot();
  });
});
