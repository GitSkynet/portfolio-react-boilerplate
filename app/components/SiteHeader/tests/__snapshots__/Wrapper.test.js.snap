// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Wrapper /> should adopt the "backgroundColor" property 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Wrapper__Wrapper
    borderColor="#bada55"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "className": "Wrapper__Wrapper-fEDTQp chtSlL",
    },
    "ref": null,
    "rendered": null,
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "className": "Wrapper__Wrapper-fEDTQp chtSlL",
      },
      "ref": null,
      "rendered": null,
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`<Wrapper /> should render and match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Wrapper__Wrapper />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "className": "Wrapper__Wrapper-fEDTQp lbOYCf",
    },
    "ref": null,
    "rendered": null,
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "className": "Wrapper__Wrapper-fEDTQp lbOYCf",
      },
      "ref": null,
      "rendered": null,
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
